### PR TITLE
Move style rules for EventTile on ThreadView from _ThreadPanel.scss to _EventTile.scss

### PR DESCRIPTION
**Pull request by luixxiul**
This PR moves style rules for `EventTile` on ThreadView from `_ThreadPanel.scss` to `_EventTile.scss` in order to:

- Increase maintainability. Having style rules for the same selector over multiple files makes it difficult to grasp the whole structure.
- Decrease chance of style inconsistencies
- Remove unexpected style cascading. As the style rules are imported alphabetically without taking the component structure into consideration, every style block can be overridden by another and is essentially vulnerable to unexpected regressions. 

|Before|After|
|---------|------|
|before|after

Signed-off-by: Suguru Hirahara <luixxiul@users.noreply.github.com>

type: task

---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

### CODE REVIEW

File: res/css/views/right_panel/_ThreadPanel.scss [hunk 1]
```diff
@@ -119,17 +119,10 @@ limitations under the License.
         }
     }
 
+    // For style rules of EventTile in a thread, see _EventTile.scss
     &.mx_ThreadView {
         max-height: 100%;
 
-        // Inside a thread timeline only
-        .mx_GenericEventListSummary {
```
Comment (luixxiul) [inline res/css/views/right_panel/_ThreadPanel.scss:126]: Moved to https://github.com/matrix-org/matrix-react-sdk/pull/8689/files#diff-bd83194d78de871bc4ad1c4e8d43102c4368481bbb90705152d3d17fea813919R1029

File: res/css/views/right_panel/_ThreadPanel.scss [hunk 2]
```diff
@@ -207,67 +200,6 @@ limitations under the License.
...
     }
 
-    &.mx_ThreadView .mx_EventTile {
-        // handling for hidden events (e.g reactions) in the thread view
-
-        &:not([data-layout=bubble]) {
-            &:hover.mx_EventTile_verified.mx_EventTile_info .mx_EventTile_line,
-            &:hover.mx_EventTile_unverified.mx_EventTile_info .mx_EventTile_line,
-            &:hover.mx_EventTile_unknown.mx_EventTile_info .mx_EventTile_line {
-                padding-inline-start: 0; // Override
```
Comment (luixxiul) [inline res/css/views/right_panel/_ThreadPanel.scss:217]: Moved to https://github.com/matrix-org/matrix-react-sdk/pull/8689/files#diff-bd83194d78de871bc4ad1c4e8d43102c4368481bbb90705152d3d17fea813919R926

File: res/css/views/right_panel/_ThreadPanel.scss [hunk 3]
```diff
@@ -207,67 +200,6 @@ limitations under the License.
...
-            }
-        }
-
-        &.mx_EventTile_info {
```
Comment (luixxiul) [inline res/css/views/right_panel/_ThreadPanel.scss:221]: Moved to https://github.com/matrix-org/matrix-react-sdk/pull/8689/files#diff-bd83194d78de871bc4ad1c4e8d43102c4368481bbb90705152d3d17fea813919R871

**Review by andybalaam — APPROVED** [review]
Good work, thank you!

**Comment by andybalaam** [comment]
The Percy failure is the same Chrome bug we've seen before. Other browsers are identical.

**Comment by luixxiul** [comment]
@andybalaam thank you very much for reviewing! I will look for possible regressions of EventTile on ThreadPanel.

**Pull request merged**